QUALITY_DECLARATION.md

```diff
@@ -50,7 +50,12 @@ All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://i
 
 ### Continuous Integration [2.iv]
 
-Currently not tested on CI.
+Pull requests must pass CI under Linux and Windows environments set in with AppVeyor and Travis-CI. Jobs are automatically trigered with each PR and the results shown in the Github repository.
+
+Current test results are shown here:
+
+[Linux (Travis CI)](https://travis-ci.org/github/ros/console_bridge)
+[Windows (Appveyor)](https://ci.appveyor.com/project/tfoote/console-bridge)
```

Comment:
It's not possible to get Appveyor under ROS or the console_bridge name?

Reply:
I'm not sure how appveyor works exactly. I believe I didn't have sufficient permissions to enable appveyor for this repo, so @tfoote did it for me.

It's way overkill, but if we switched to GitHub actions, I think that would get rid of `tfoote`'s name in the URL. 😅